core/gatsby-theme-docz/src/components/Playground/index.js

```diff
@@ -79,14 +99,22 @@ export const Playground = ({ code, scope, language }) => {
             </button>
           </div>
         </div>
+        {showingCode && (
+          <Wrapper height={editorHeight}>
```

Comment:
Am i correct in the assumption that this was done to solve this issue #1271 ?

Because now that you are wrapping the component in a iframe the encapsulation of the editor should not be needed any more. Am i missing something? :)

Reply:
> Am i correct in the assumption that this was done to solve this issue #1271 ? 
Yep that's exactly why ! Initially wanted to only wrap the preview.

I'm wrapping the preview and the editor each in an iframe, not wrapping the whole Playground with an iframe so it should be necessary if I'm not mistaken.

Reply:
I think the issue was that the css from the component inside the preview was styling the code area of the preview. This should not be possible any more after the component preview is wrapped in a iframe.